capa/features/freeze/__init__.py

```diff
@@ -300,6 +305,7 @@ class Config:
 class Freeze(BaseModel):
     version: int = 2
     base_address: Address = Field(alias="base address")
+    sample_hashes: SampleHashes
```

Comment:
do we need these in the freeze format?

Reply:
I added it so that the `NullStaticFeatureExtractor` could implement the `get_sample_hashes()` base method. Displaying the hashes for the `FREEZE` format as well seems logical to me, plus getting rid of that would create a corner case in main.py (collect_metadata()) that i feel like is unnecessary